app/src/userpages/components/Avatar/index.jsx

```diff
@@ -38,14 +28,9 @@ const Avatar = ({
         {!linkToProfile && (
             <AvatarCircle name={user.name} imageUrl={user.imageUrlLarge} className={styles.avatarCircle} uploadAvatarPlaceholder />
         )}
-        {!editable && (
```

Comment:
Editing removed from `Avatar` component. It's only used in the Profile section and the modal is also moved there.